src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -5076,4 +5076,7 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="ERR_DefaultInterfaceImplementationModifier" xml:space="preserve">
     <value>The modifier '{0}' is not valid for this item in C# {1}. Please use language version {2} or greater.</value>
   </data>
+  <data name="ERR_ImplicitImplementationOfNonPublicInterfaceMember" xml:space="preserve">
+    <value>'{0}' does not implement interface member '{1}'. '{2}' cannot implicitly implement '{1}' because '{1}' is not public.</value>
```

Comment:
I think there is a value in the second sentence. How about I change it to "'{2}' cannot implicitly implement non-public member" ?